language-server/modules/langserver-core/src/test/resources/completion/expression_context/source/new_expr_ctx_source27.bal

```diff
@@ -0,0 +1,5 @@
+import ballerina/module1;
+
+function testFunction() {
+    final module1:Client clientEP = check new("http://example.com", );
```

Comment:
I don't think this is a good example, because the `module1:Client` have only 1 parameter (url) in its `init` method. We need more parameters.

We can write a client class with included records as parameters.